docs/source/Snapshot.rst

```diff
@@ -0,0 +1,29 @@
+.. _snapshot_header:
+
+Snapshot
+=============
+
+===========
```

Comment:
It's annoying, but the docs throw a warning unless these are as long as the title. Can you make them longer?

Reply:
Sure thing